org/wildfly/undertow/listener/capability.adoc

```diff
@@ -0,0 +1,30 @@
+NAME: org.wildfly.undertow.listener
+
+DESCRIPTION: Provides an Undertow listener
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: false
+
+SUPPORTS RUNTIME ONLY: false
+
+SERVICE PROVIDED:
+
+  CLASS: org.wildfly.extension.undertow.ListenerService<T>
```

Comment:
This can't be the declared service type. What's needed isn't the name of the class implementing org.jboss.msc.service.Service, it's the type returned by that class' getValue() method. In this case that is "T" and that's not a valid value for a capability contract.